src/components/misc/RowLink.vue

```diff
@@ -1,19 +1,32 @@
 <template>
-    <a :href="hrefWithDefault" class="row-link d-flex flex-grow-1 align-items-center gap-7 rounded">
+    <a class="row-link d-flex flex-grow-1 align-items-center gap-7 rounded" data-bs-toggle="dropdown" @click="showActions">
         <img :src="iconB64Svg" :alt="`${text} icon`">{{ text }}
-        <ChevronRight class="ms-auto" />
+        <ChevronDown v-if="actionsVisible" class="ms-auto" />
+        <ChevronRight v-else class="ms-auto" />
     </a>
+    <ul v-if="actionsVisible">
```

Comment:
Now a "Link" becomes more of a collapsible. 
Either 
- change the name of this component 
- or move this part to another component.